helix-term/src/commands.rs

```diff
@@ -2216,6 +2216,7 @@ fn buffer_picker(cx: &mut Context) {
         path: Option<PathBuf>,
         is_modified: bool,
         is_current: bool,
+        used_at: std::time::Instant,
```

Comment:
Could we call this "focused_at" or "accessed_at"? "used_at" lacks some context about what in the document is used

Reply:
done